probe/docker/container.go

```diff
@@ -215,6 +223,7 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 		ImageID:          c.container.Image,
 		ContainerIPs: strings.Join(append(c.container.NetworkSettings.SecondaryIPAddresses,
 			c.container.NetworkSettings.IPAddress), " "),
+		ContainerLabels: string(labels),
```

Comment:
After discussion with @peterbourgon, we'd prefer if this was encoded as multiple entries in the nodemetadata map - where the key in the map is of the form docker_container_label_<label key>.